### PR TITLE
Add username to get_object_perms shortcut

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,15 @@ All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+- "Add ``username`` to ``get_object_perms`` shortcut
```
Comment (dblenkus) [inline docs/CHANGELOG.rst]: Please describe a user-facing change, not the implementation details, e.g. `Add username to permission objects on the API`. And move it under `Added`.

Comment (JureZmrzlikar) [reply docs/CHANGELOG.rst]: Done.

Comment (JureZmrzlikar) [reply docs/CHANGELOG.rst]: If this fix makes it OK, please merge.

**Comment by JureZmrzlikar** [comment]
@dblenkus Can we merge this?

**Pull request merged**